securedrop/journalist_app/admin.py

```diff
@@ -34,6 +36,11 @@ def manage_config():
             static_filepath = os.path.join(config.SECUREDROP_ROOT,
                                            "static/i/logo.png")
             f.save(static_filepath)
+
+            with Image.open(static_filepath) as im:
```

Comment:
If someone gives us a PNG or JPG that Pillow doesn't understand, the use will get the generic 500 error page. We should try/catch this to and `flash` something like "We couldn't process that image. Try another one."

Reply:
I did not write any exception handling here because I think Flask handles the type of images, and only JPG/JPEG/PNG make through. But that is a good point, `Pillow` may encounter something unexpected. I'll update the code.

Reply:
I'm rather sure that the `FileAllowed` validator only checks the extension, and it's possible someone has a valid PNG that Pillow doesn't understand. We should try to catch these cases.